tests/run_tests/apps_inversion_test.py

```diff
@@ -160,11 +161,14 @@ def test_mag_inversion(tmp_path: Path):
                 reference_declination_const=None,
             )
 
-    filename = next(tmp_path.glob("MagneticVectorInversion_*.geoh5"))
-    with Workspace(filename) as workspace:
-        assert len(workspace.get_entity(app.params.data_object)) == 1
-        assert len(workspace.get_entity(app.params.mesh)) == 1
-        assert len(workspace.get_entity(app.params.topography_object)) == 1
+    filename = next(tmp_path.glob("MagneticVectorInversion_*.json"))
+    ifile = InputFile.read_ui_json(filename)
+    with ifile.data["geoh5"].open():
+        data = ifile.data["data_object"]
+        assert isinstance(data, Points)
+        assert data.n_vertices == 418
+        assert ifile.data["mesh"] is not None

```

Comment:
maybe test further than just is not None?
To verify the obejcts are preserved/transformed the right way?

Reply:
Fair enough